Multisig/UI/ClaimToken/Select Guardian/GuardianTableViewCell.swift

```diff
@@ -31,11 +31,13 @@ class GuardianTableViewCell: UITableViewCell {
             hideAddress: false,
             label: guardian.name,
             imageUri: guardian.imageURL,
+            placeholderImage: "ico-no-delegate-placeholder",
             badgeName: nil,
             prefix: nil)
 
         addressInfoView.copyEnabled = false
         
         descriptionLabel.text = guardian.reason
+        layoutIfNeeded()
```

Comment:
why would it be needed? What makes it necessary to call it explicitly?

Reply:
Here we are in reusable view, when you replace it with longer text the size of cell is not adjusted without explicit call